.buildkite/build.sh

```diff
@@ -1,10 +1,19 @@
-#!/bin/sh
+#!/bin/bash
 set -e
+image="totvslabs/presto"
 
-echo "~~~ build"
+# echo "~~~ build"
```

Comment:
💅 This echo makes "text blocks" on Buildkite... Maybe keeping it is better.

Reply:
Typo.. fixing